js/scripts.js

```diff
@@ -0,0 +1,10 @@
+var femaleNames = ['Asia', 'Kasia', 'Ola', 'Jola'];
+var maleNames = ['Piotrek', 'Marek', 'Arek', 'Jarek'];
+var allNames = femaleNames.concat(maleNames);
+var newName = ['Marian'];
+
+if (allNames.indexOf(newName) === -1) {
+	var newName = allNames.push('Marian');
```

Comment:
instead of pushing 'Marian', push that data what you were checking in previous line